types/django-bananas/index.d.ts

```diff
@@ -0,0 +1,222 @@
+// Type definitions for django-bananas 4.0
+// Project: https://github.com/5monkeys/django-bananas.js
+// Definitions by: Elias Sjögreen <https://github.com/eliassjogreen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+import * as React from 'react';
+
+interface AppProps {
+    api: ApiSetting;
+    pages: (route: string) => any;
+    prefix?: string;
+    logLevel?: 'INFO' | 'DEBUG' | 'WARN' | 'ERROR' | 'OFF';
+
+    layout?: 'horizontal' | 'vertical';
+    permanent?: boolean;
+    collapsed?: boolean;
+    dense?: boolean;
+
+    title?: string;
+    branding?: string;
+    version?: string;
+    logo?: React.ReactNode | string | boolean;
+    nav?: string[] | Record<string, NavItem | NavAppItem>;
+
+    theme?: any;
+    pageTheme?: any;
+    loginForm?: (...args: any) => any;
+    editableSettings?: boolean;
+    customizeContext?: (context: any) => any;
+    customiseUser?: (user: UserInterface) => UserInterface;
```

Comment:
```suggestion
```
where this one came from? (the spelling is different)

Reply:
Oops